pypdf/_writer.py

```diff
@@ -122,7 +122,7 @@
 )
 
 OPTIONAL_READ_WRITE_FIELD = FieldFlag(0)
-ALL_DOCUMENT_PERMISSIONS = UserAccessPermissions((2**31 - 1) - 3)
+ALL_DOCUMENT_PERMISSIONS = UserAccessPermissions.all()
```

Comment:
Uh, nice! I didn't know that!

Reply:
Even if it would have been provided, it would not work as we have to subtract the first two bits (the `- 3` part) as they are reserved and should always be zero.